spring-xd-integrations.tex

```diff
@@ -1,68 +1,88 @@
 \section{Integrations}
 Spring XD integration modules are Message Endpoints \cite{enterprise-integration-pattern-message-endpoint} 
-that are responsible for sending data to and receiving data from external agents.  
-There are 3 types of message endpoints: Source, Sinks and Batch Jobs.  
-A source is the entry point for data into the stream. A sink is the module that dispatches 
-the stream's results to an external agent or storage system.  Batch Jobs may be used to 
-execute batch processing steps on a set of data or process.  
-Spring XD offers a suite of 23 sources (file, jdbc, Mongo, HDFS, \ldots), 24 sinks
- (file, jdbc, Mongo, HDFS, \ldots) and 9 jobs
-(filepollhdfs, sparkapp, sqoop, \ldots)  that are ready to use at XD startup.  
-If an existing module does meet the needs of for the given use case a user may create a 
-custom integration module.\par
+that are responsible for sending data to and receiving data from external applications.
```

Comment:
The Message Endpoint pattern uses the term "application" instead of "agent" - thus my suggestion that we adopt this term.